### DISCUSSION TITLE
2023 Weekly Community Updates

### DISCUSSION THREAD

**Discussion by aleesteele** [the-turing-way/the-turing-way#3018] [Announcements]
In the #ttw-core-team team channel on Slack (currently private), I have been posting weekly updates related to the project (talks we've been invited to, upcoming events, and pending submissions). In an effort to make this work more open and transparent for the wider community, outside of the organisers - I am starting the process of adding in these weekly reviews here.

Events calendar: https://docs.google.com/spreadsheets/d/1C-VZvmFL4PnSBsv_G9ZD3dwjIYLno3NyL7oHvbplnWs/edit#gid=0 

Pad for community attendees: https://pad.sfconservancy.org/p/ttw-event-calendar-2023

**Comment by aleesteele** [comment]
## Archive

### Archiving presentations/events from 2023 so far
- [x] 7 February - Data Science Perspectives (Newcastle UK) - #2952
- [x] 7-8 February - State of Open Con (London, UK) 
- [x] 9 February - Open Data Day / TPS Coffee Chats (online) - no issue
- [x] 10 March - Data Hazards Keynote (in person, London, UK) - no issue 
- [x] 10 March - DHNordicBaltic Conference - no issue
- [x] 14 March - Maintenance as a Turing Way (Berlin, Germany) - #2993 
- [x] 21 March - Researcher to Reader Conference (London) - no issue
- [x] 24 March - RDA Plenary (Stockholm, Sweden)- no issue
- [x] 21-25 March - MozFest (online) -  no issue
- [x] 21-22 March - AI-UK Turing Way exhibit (London, UK) - no issue
- [x] 22 March- Git Good Workshop @ AI-UK (London, UK) - #3006 
- [x] 31 March - Fireside Chat - #2982 
- [x] 13 April - BCSCWomen Lovelace Colloquium (Sheffield, UK) - no issue
- [x] 18-20 April - FORCE 2023 (Online) - no issue 
- [x] 17-19 April - CZI Meeting (Buenos Aires, Argentina)
- [x] 19 April - Climate Informatics Panel (Cambridge, UK, online)
- [x] 19-20 - csv,CONF (Buenos Aires, Argentina)
- [x] 20 April - Julia Silge visit (London, UK)
- [x] 24 April - UCL Open Science and the Case for Social Justice (London, UK)
- [x] 27 April - TTW Case study workshop RSS/UKRM Repro Roundtable (online)
- [x] 2-4 May - Collaborations Workshop (Manchester, UK) - see #collaborations-workshop-23 on slack, two workshops given
- [x] 5 May - Fireside Chat (online)
- [x] 16 May - Edinburgh Open Research Conference 2023 (Edinburgh, UK)
- [x] 17 May - TPS Team Meeting (online)
- [x] 18 May - TPS Coffee Chat (London, UK & online)
- [x] 19 May - EDS Book + Climate Informations "Chat with Experts" (online)
- [x] 25 May - OSPO++ Presentation (online)
- [x] 1 June - Lunch & Learn (internal ATI event) 
- [x] 12-14 - Moving frontiers of the demos (Geneva, CH)
- [x] 19-20 June - Data Justice Conference (Cardiff, UK)
- [x] 21 June - UN OSPOs for Good (NY, USA, hybrid)
- [x] 28 June - ACM-REP (CA, USA, hybrid)
- [x] 27-29 June - Open Science EU (online)
- [x] 3-9 July - London Data Week (London, UK, in person & online)
- [x] 12 July - QQR @ Turing (London, UK, in person)
- [x] 21 August - Widening Participation Workshop (London, UK, in person)
- [x] 4 September - OpenFest (Sheffield, UK, online)
- [x] 21 September - Festival of Hidden REF (Bristol, UK)
- [x] 25-27 September - Open Science Fair (Madrid, Spain)
- [x] 26-28 September - deRSE Unconference (Dornburger Schlösser, Jena, Germany)
- [x] 17-18 October - NHS-R Community Conference (Birmingham, UK & online)
- [x] 30 October - British Library, Open & Engaged 2023
- [x] 8-10 November - Munin Conference on Scholarly Publishing (Tromsø, Norway)
- [x] 20-24 November - EMBL-EBI Genome Bioinformatics 2023 course (Cambridge, UK)

### Archiving team events
- [x] 10 March - Core Team Quarterly Meeting 
- [x] 29 March - Infrastructure Meeting - organised in #2690 

### Archiving applications submitted
- [x] FOSDEM (accepted) 
- [x] Metascience 2023 (rejected)
- [x] RightsCon (rejected)
- [x] Data Justice 2023 (accepted)
- [x] FOSS Backstage (accepted)
- [x] FORCE11 (accepted)
- [x] MozFest 2023 (accepted)
- [x] Collaborations Workshop (accepted)
- [x] Edinburgh Open Research Conference (accepted)
- [x] Open Research EU (accepted)
- [x] Open Science Fair (accepted)

### Invited talks
- [x] UCL Talk
- [x] Climate informatics
- [x] OSPO++
- [x] HiddenREF Festival
- [ ] NHS-R Community (Birmingham, UK, 17-18 Oct)

### Themed weeks
- Open Data Day:
- Open Access Week: October 23 - 29, 2023: https://www.openaccessweek.org/
- Big Team Science: 23-25 October: bigteamscienceconference.github.io/

**Reply by AlexandraAAJ** [reply]
Hi Anne, would it be possible to add in this list, the names of the team members who facilitated/gave the workshop/talks?

**Reply by AlexandraAAJ** [reply]
I mean for the future ones

**Reply by aleesteele** [reply]
Absolutely @AlexandraAAJ !

**Comment by aleesteele** [comment]
## 17-04-2023

### Upcoming events
- [x] FORCE 2023 (18-20 April)
- [x] CZI Meeting (17-19 April)
- [x] Climate Informatics (20-21 April)
- [x] csv,CONF (19-20 April)
- [x] Julia Silge visit (20 April): organising 

### Working Groups & Team events
- [ ] In progress - Follow-up meeting for infrastructure team 
- [x] In progress - Translation & Localisation team support

### Meeting summaries
- Review
- Review
- Review

**Comment by aleesteele** [comment]
## 27-04-2023

### Event Review
- [x] 24 April - UCL Open Science and the Case for Social Justice (London, UK): went well! lots of engagement with TTW project, we spoke about Research Infrastructure Roles and Translation/Localisation
- [x] 27 April - TTW Case study workshop RSS/UKRM Repro Roundtable (online)

### Working Groups & Team events
- [ ] Accessibility Working Group - first meeting 

### Meetings
- 

### Links

**Comment by aleesteele** [comment]
## 05-05-2023

### Event Review
- [x] 2-4 May - Collaborations Workshop (Many different presentations by TTW folks!)
- [x] 5 May - Fireside Chat on Open Science

### Working Groups & Team events
- N/A

### Meetings
- N/A

### Links

**Comment by aleesteele** [comment]
## 12-05-2023

### Event Review
- [x] 11 May - Github Workshop (Pre-Book Dash)

### Working Groups & Team events
- [x] Infrastructure WG Meeting

### Meetings
- N/A

### Links

**Comment by aleesteele** [comment]
## 19-05-2023

### Event Review
- [ ] 16 May - Edinburgh Open Research Conference 2023 (Edinburgh, UK)
- [ ] 19 May - EDS Book + Climate Informations "Chat with Experts" (online)

### Working Groups & Team events
- [ ] 

### Meetings
- N/A

### Links

**Comment by aleesteele** [comment]
## 31-05-2023

Flagging some materials I had shared within my team this week. After attending Collaborations Workshop, there were a couple of emerging themes. 


Just my view, but most of the information you see around these topics focus on either the labor behind open source and or different types of business models. For example: 
- The most famous "open source" dichotomy is the cathedral vs bazaar model, which essentially pits a closed development process w/ open release against a open development w/ open release process: https://en.wikipedia.org/wiki/The_Cathedral_and_the_Bazaar
- 'Roads & Bridges' by Nadia Eghbal has a great section called "sustaining digital Infrastructure" which lists a bunch of business models as well: Bounties, Services, Paid Licenses, Sponsors/Donors, Corporate Sponsorship (essentially overlapping or expanding what @Bastian Greshake Tzovaras already shared!).  [not attached here!]
- This medium article also highlights different types in a more business-friendly format & with charts: https://insights.blossomcap.com/successful-open-source-business-models-2709e831e38a

However, the above may not be as relevant or perhaps too general for what we're talking about right now: because support doesn't just look like funding £$, but also institutional structures that enable it.

Along these lines, I've increasingly seen three institutionalised models gaining a lot of steam & publicity recently, as well as securing funding at massive scale: Open Source Programme Offices (OSPOs), Inner Source, and Digital Public Goods.

- **Open Source Programme Offices (OSPOs)**: Open source programme offices are getting a lot of traction in US & Europe right now, with a couple of offices being incubated in EU universties, and some major open source players like Linux/Eclipse getting involved
- --> "The Cathedral, the Bazaar & the Coffeehouse": Saw this talk at FOSS Backstage that argued for them, building off of the cathedral vs bazaar idea: https://program.foss-backstage.de/fossback23/talk/8YHTY9/
- **Inner Source**: Inner source also seems to be gaining traction as a means of disentangling open ways of working WITHIN a group or institution from open source as a public process (i.e. enabling the cathedral method when necessary when the bazaar method seems to be the moral good right now!)
- --> "Inner Source 101": An intro talk, also from FOSS Backstage: https://www.youtube.com/watch?v=zlMeNA8p7vU&list=PLq-odUc2x7i9kfJWH6yBsHuJasukhluUi&index=1
- **Digital Public Goods / Public code**: The digital public goods & digital public infrastructure narrative has been around for a while now, but a few actors seem to be emerging as the leads in this space, like the Foundation for Public Good & the Digital Public Goods Alliance. Also getting a lot of traction of the EU/USA level.
- ---> "How the Standard for Public Code can help you collaborate over time & space": Yet another FOSS Backstage intro! I did hear about these before this conference! https://www.youtube.com/watch?v=SKl3BX9kAbY
- Also attached a report from the Standard for Public Code that aims to make this process (the fact that it splits between policy makers, management, and designers & developers is great!)

**Comment by Dorianteffo** [comment]
Hello, my name is Dorian, and i am a junior data analyst. 
Here is my medium https://medium.com/@dorianteffo 
Don't hesitate to leave me comment thanks.

**Comment by aleesteele** [comment]
Closing this – as we're now in 2024!
